README.md

```diff
@@ -10,6 +10,12 @@ MacDown is an open source Markdown editor for OS X, released under the MIT Licen
 
 Visit the [project site](http://macdown.uranusjr.com/) for more information, or download [MacDown.app.zip](http://macdown.uranusjr.com/download/latest/) directly from the [latest releases](https://github.com/MacDownApp/macdown/releases/latest) page.
 
+## Install
+
+[Download](http://macdown.uranusjr.com/download/latest/), unzip, and drag the app to Applications folder. MacDown is also available through [Homebrew Cask](https://caskroom.github.io/):
+
+    brew cask install macdown
+
```

Comment:
Now reading this again, the instruction could really use some information about the “unidentified developer” warning when you open the app. But that’s for another day.